packages/hint/src/lib/@types/jsdom-lib-old-api.d.ts

```diff
@@ -0,0 +1,2 @@
+// Let the old jsdom API be type `any` until removed.
+declare module 'jsdom/lib/old-api';
```

Comment:
I think we can remove this one now that we've migrated to jsdom 12, right?

Reply:
Yep, done.